gradle/libs.versions.toml

```diff
@@ -27,7 +28,7 @@ jakarta-annotation-api = "3.0.0"
 jakarta-transaction-api = "2.0.1"
 jakarta-validation-api = "3.1.0"
 jakarta-xml-bind-api = "4.0.2"
-java-stellar-sdk = "0.44.1"
+java-stellar-sdk = "1.0.0-beta1"
```

Comment:
This should be `1.0.0-rc0`

Reply:
I have already updated it to rc0. I expect to release the stable version of the stellar sdk next week. At that time, we can release the new wallet sdk and then update this repo.